views/publication/summary.layout.gohtml

```diff
@@ -1,134 +1,137 @@
+                                <i class="if if-download if--small if--success"></i>
                                 {{else}}
-                                <i class="if if-ghent-university if--small if--muted"></I>
+                                <i class="if if-ghent-university if--small if--primary"></i>
                                 {{end}}
-                                <span class="c-subline text-truncate ps-2">{{.Loc.Get (print "publication_file_access_levels_after_embargo." $mainFile.AccessLevelAfterEmbargo)}} from {{$mainFile.EmbargoDate}}</span>
-                            </div>
+                                {{.Loc.Get (print "publication_file_access_levels_after_embargo." $mainFile.AccessLevelAfterEmbargo)}} from {{$mainFile.EmbargoDate}}
+                            </span>
+                            {{end}}
+                            {{else if not .Publication.Extern}}
+                            <span class="c-subline ps-2 me-3 pe-3">
+                                <a href="{{.PublicationURL|querySet "show" "files"}}" class="badge badge-danger-light badge-sm ps-3">
+                                    <i class="if if-edit if--small"></i>
+                                    <span class="badge-text">Add document type: full text</a>
+                                </a>
+                            </span>
                             {{end}}
-                        {{else if not .Publication.Extern}}
-                        <div class="c-subline text-nowrap me-3 pe-3 my-2">
-                            <i class="if if-error-circle if--error"></i>
-                            <span class="ps-2 text-muted">Full text missing</span>
                         </div>
-                        {{end}}
-                    </div>
-                    <div class="bc-toolbar-right">
-                        {{block "summary_actions" .}}{{end}}
-                    </div>
-                </div>
-                <h4 class="list-group-item-title">
-                    {{if .User.CanCurate}}
-                    {{if .Publication.Title}}
-                        {{.Publication.Title}}
-                    {{else}}
-                        Untitled record
-                    {{end}}
-                    {{else}}
-                    <a href="{{.PublicationURL}}">
-                    {{if .Publication.Title}}
-                        {{.Publication.Title}}
-                    {{else}}
-                        Untitled record
-                    {{end}}
-                    </a>
-                    {{end}}
-                </h4>
-
-                <div class="c-meta-list c-meta-list-horizontal">
-                    {{range .Publication.SummaryParts}}
-                    <span class="c-meta-item">{{.}}</span>
-                    {{end}}
-                </div>
+                        <h4 class="mb-0">
+                            {{if eq .UserRole "curator"}}
```

Comment:
Question to @nics @netsensei 

 Should we use `.User.CanCurate` or `eq .UserRole "curator"`? I see a mix of both happening,
and they indeed have the same effect. Just an aesthetic remark.

Reply:
they're not the same, one tests if you have permission, the other tests your current role

e.g.
.User.CanCurate and eq .UserRole "user"
could be a reviewer looking at the (blue) user perspective